mvp-spec/prebid-implementation.md

```diff
@@ -188,7 +188,7 @@ pbjs.setConfig({
 });
 ```
 
-The PAF submodule will create a transaction for each adUnit and append the transactions
+The PAF submodule will create a transaction_id for each adUnit and append the transaction_ids
 to each bidRequest under the ortb2imp field as follows.
 
 <!--partial-begin { "files": [ "prebid-ortb2imp.json" ], "block": "json" } -->
```

Comment:
I don't understand how mvp-spec/partials/prebid-ortb2.json is modified, but prebid-ortb2imp.json gets updated in here?

Reply:
We included the `paf` object in an `ext` object to respect the specs of ortb.